test/ciphers/simple_sub_01.cipher

```diff
@@ -0,0 +1,2 @@
+abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ
+EprsJlTumxyQVIXeWoRUYcgOtKMidBvGFaNbqLkfSwHzAZhPDCjn
```

Comment:
When I tested tesseract with some handwriting it had some trouble telling apart uppercase and lowercase. I'm thinking maybe we should normalize the inputs to either of the cases before enconding/decoding them. This would mean that uppercase letters wouldn't be able to be transposed with lowercase and viceversa.

Reply:
i have pushed [385ee13](https://github.com/ransomw/the-benevolent-dictators/pull/17/commits/385ee1389ad5e60d56af289dd03ca35a9a3a2f5a) to address what i think the suggestion is here